packages/diagrams/frontend/sirius-components-diagrams/src/renderer/layout/useDistributeElements.ts

```diff
@@ -246,7 +249,11 @@ export const useDistributeElements = (refreshEventPayloadId: string): UseDistrib
     (selectedNodes: Node[], selectedNodeIds: string[], refNode: Node): Node[] => {
       const largestHeight: number = selectedNodes.reduce((height, node) => Math.max(height, node.height ?? 0), 0);
       return getNodes().map((node) => {
-        if (!selectedNodeIds.includes(node.id)) {
+        if (
```

Comment:
The formatting seems strange.
There is nothing after the `if (`.
Could you check it please?

Reply:
It seems to be the correct formatting